src/app/zap-templates/zcl/data-model/chip/energy-evse-cluster.xml

```diff
@@ -0,0 +1,235 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2023 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<configurator>
+  <domain name="Energy Management"/>
+
+  <enum name="EvseStateEnum" type="ENUM8">
```

Comment:
```suggestion
  <enum name="EvseStateEnum" type="enum8">
```
and similar for the other ones.

Reply:
Changed to `enum8`

Reply:
>  We were following convention used on other clusters.

The convention changed while you were working on the PR...

Reply:
>  We were following convention used on other clusters.

The convention changed while you were working on the PR @jamesharrow